### PR TITLE
feat: Enable Net Analyzer

### PR DESCRIPTION
**Pull request by workgroupengineering**

### CODE REVIEW

**Review by kekekeks — APPROVED** [review]

File: src/XamlX/IL/RecordingIlEmitter.cs [hunk 1]
```diff
@@ -197,11 +197,11 @@ public override string ToString()
             {
                 var i = Instructions[c];
                 sb.AppendFormat("{0000}", c);
-                sb.Append(": ");
+                sb.Append(':').Append(' ');
```
Comment (maxkatz6) [inline src/XamlX/IL/RecordingIlEmitter.cs]: CA1834 doesn't prohibit Append of strings longer than one character.

Comment (workgroupengineering) [reply src/XamlX/IL/RecordingIlEmitter.cs]: You have right, but this change is made to avoiding allocation of ": " in  intern string pool.

**Comment by workgroupengineering** [comment]
What other changes do I need to make?

**Review by maxkatz6 — APPROVED** [review]

**Comment by maxkatz6** [comment]
cc @kekekeks

File: .editorconfig [hunk 2]
```diff
@@ -1,2 +1,60 @@
 # This file is the top-most EditorConfig file
-root = true
\ No newline at end of file
+root = true
+[*.cs]
+
+# CS0162: Remove unreachable code
+dotnet_diagnostic.CS0162.severity = error
```
Comment (maxkatz6) [inline .editorconfig:6]: Can it be warning instead? It was proven to be annoying during debugging.

Comment (maxkatz6) [inline .editorconfig]:
```suggestion
dotnet_diagnostic.CS0162.severity = warning
```

Can it be warning instead? It was proven to be annoying during debugging.

**Pull request merged**